src/app/mod.rs

```diff
@@ -14,9 +14,11 @@ pub(crate) mod replicasets;
 pub(crate) mod replication_controllers;
 pub(crate) mod secrets;
 pub(crate) mod statefulsets;
+pub(crate) mod storageclass;
 pub(crate) mod svcs;
 mod utils;
 
+use crate::app::storageclass::KubeStorageClass;
```

Comment:
move it to existing import grouping below in line 40. If you are using VSCode it should do that when you format